docs: alternative way to read a projected service account token (#875)
snyk/kubernetes-monitor: snyk-monitor/README.md

--set excludedNamespaces="{kube-node-lease,kube-public,local-path-storage,some_namespace}"
```

## Using EKS without assigning an IAM role to a Node Group

> If you do not want to assign an IAM role to a Node Group, there is an alternative way to read a projected service account token with nobody by using securityContext.fsGroup:

Reviewer: ```suggestion
If you do not want to assign an IAM role to a Node Group, you can use the IAM role for Service Accounts and configure the snyk-monitor as follows:
```

> - Setting an IAM role for a service account : [IAM role for a service account Setting](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html)

Reviewer: ```suggestion
- Setting an IAM role for a service account: [IAM role for a Service Accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html)
```

> - Modifying the fsGroup of the mounted EKS credentials in snyk-monitor

Reviewer: ```suggestion
- Modify the fsGroup of the mounted EKS credentials in snyk-monitor to the user `nobody` (uid `65534`)
```

> - Annotated the snyk-monitor service account with the IAM role

Reviewer: ```suggestion
- Annotate the snyk-monitor service account with the IAM role
```

```shell
helm upgrade --install snyk-monitor snyk-charts/snyk-monitor \
  --namespace snyk-monitor \

>   --set securityContext.fsGroup="65534" \

Reviewer: ```suggestion
  --set securityContext.fsGroup=65534 \
```

Reviewer: We should also add this option:

```shell
--set volumes.projected.serviceAccountToken=true
```

>   --set rbac.serviceAccount.annotations={eks.amazonaws.com/role-arn: <iam role name> }

Reviewer: ```suggestion
  --set rbac.serviceAccount.annotations."eks.amazonaws.com/role-arn"="<iam role name>"
```
